app/javascript/components/student/tracks-list/Header.jsx

```diff
@@ -1,14 +1,28 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import pluralize from 'pluralize'
+import { Loading } from '../../common/Loading'
 
-export function Header({ data, query = {} }) {
+export function Header({ latestData, query = {} }) {
```

Comment:
I don't get the reason for this rename inside this component. As far as the Header is concerned is this not always it's `data`?

Reply:
Ah yes. I'll rename it back to data.